server/controller/main.go

```diff
@@ -82,7 +82,7 @@ func Authenticated(w http.ResponseWriter, r *http.Request) bool {
 		return false
 	}
 
-	return true
+	return util.IsSystemAdmin(userID)
```

Comment:
1/5 suggest renaming `s/Authenticated/IsAdmin` and `s/RequiresAuth/RequiresAdmin`